rrmngmnt/host.py

```diff
@@ -232,8 +233,8 @@ def executor(self, user=None, pkey=False, sudo=False):
             )
             ef = copy.copy(self.executor_factory)
             ef.use_pkey = pkey
-            return ef.build(self, user, self.sudo)
-        return self.executor_factory.build(self, user, sudo=self.sudo)
+            return ef.build(self, user, self.sudo, disabled_algorithms=disabled_algorithms)
```

Comment:
```suggestion
            return ef.build(self, user, self.sudo, disabled_algorithms)
```
keep the same format.